srcpkgs/python3-loguru/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'python3-loguru'
+pkgname=python3-loguru
+version=0.6.0
+revision=1
+wrksrc="loguru-${version}"
+build_style=python3-module
+short_desc="Library which aims to bring enjoyable logging in Python"
+maintainer="oliver <oliver@tuta.io>"
+license="MIT"
+homepage="https://github.com/Delgan/loguru"
+distfiles="${PYPI_SITE}/l/loguru/loguru-$version.tar.gz"
+checksum=066bd06758d0a513e9836fd9c6b5a75bfb3fd36841f4b996bc60b547a309d41c
+
+post_install() {
+	vlicense LICENSE
+}
+
+do_check() {
+	python3 setup.py test
```

Comment:
this is already the default behaviour:

https://github.com/void-linux/void-packages/blob/2859518608b4f682abd4afb6b9f8c2dce01a0523/common/build-style/python3-module.sh#L40-L41

Reply:
but in this case you'd want to add `checkdepends="python3-pytest"` because the tests in this package are run with pytest

Reply:
Thanks - I really appreciate your comments and insight. 
Is there an appropriate way to test this before amending the PR?  The original template compiled perfectly in a freshly created xbps-src environment but then failed the build checks.

Does it even need a do_check{} section?

Reply:
running xbps-src pkg and xbps-src check would be how I'd test building the template